css/at-rules/media.json

```diff
@@ -1605,26 +1605,11 @@
               },
               "firefox": [
                 {
-                  "version_added": "49",
```

Comment:
Can you help me understand the data that's dropped here (i.e., commit 4468d5f261f995759c00af19d4339149d45576a5)? The existing data seems approximately right (i.e., you used to have to enable a pref to get this feature)? Was there something wrong with this data after all?